packages/indexer-native/package.json

```diff
@@ -59,12 +41,5 @@
     "jest": "<30.0.0-0",
     "prettier": "3.0.3"
   },
-  "binary": {
```

Comment:
We may want to keep this in since it is used to configure our binary build workflow.